docs/hopsworksai/azure/getting_started.md

```diff
@@ -284,14 +287,40 @@ Click on *Save* (7).
   </figure>
 </p>
 
-## Step 3: Deploying a Hopsworks cluster
+## Step 3: Adding a ssh key to your resource group
+
+When deploying instances Hopswroks.ai install a ssh key on them in case you would need to access directly to the machines. For this purpose you need to add a ssh key to your resource group.
```

Comment:
When deploying clusters, Hopsworks.ai installs a ssh key on the cluster's instances so that you can access them if necessary. For this purpose you need to add a ssh key to your resource group.